DSCResources/MSFT_xFirewall/MSFT_xFirewall.psm1

```diff
@@ -658,9 +658,9 @@ function Get-FirewallRuleProperty
         Write-Verbose "Get-FirewallRuleProperty: Get only [$Property] property"
 
         return &(Get-Command "Get-NetFirewall$Property")  -AssociatedNetFireWallRule $FireWallRule
-    }    
+    }
 }
 
 #endregion
 
-Export-ModuleMember -Function *-TargetResource
+Export-ModuleMember -Function *-TargetResource
```

Comment:
Please end a file with a newline [See item 5 in style guidelines](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md)